h2o-py/h2o/model/model_base.py

```diff
@@ -1251,15 +1251,15 @@ def _get_metrics(o, train, valid, xval):
             metrics["train"] = output["training_metrics"]
         return metrics
 
-    @deprecated_params({'save_to_file': 'save_plot_path'})
-    def partial_plot(self, data, cols=None, destination_key=None, nbins=20, weight_column=None,
+    @deprecated_params({'data': 'frame', 'save_to_file': 'save_plot_path'})
+    def partial_plot(self, frame, cols=None, destination_key=None, nbins=20, weight_column=None,
```

Comment:
How about adding frame as an input as well?  frame=None?  Yes, you will have to do some processing to decide where the data is from data or frame but this won't break people's old code if they do data=train_frame?

Reply:
@wendycwong that's what the `@deprecated_params({'data': 'frame', ` part does, it translates `data` keyword arg into a `frame` arg and it also handles the deprecation warnings automatically, including the possibility of using both old and new param… this is fully tested already, cf. `pyuniy_deprecations.py`